app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancerProperties.html

```diff
@@ -54,6 +54,19 @@ <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
         <validation-error message="Detail can only contain letters, numbers, and dashes."></validation-error>
       </div>
     </div>
+
+    <!--adding support for public facing elb -->
+    <div class="form-group">
```

Comment:
Let's add an `ng-if` to conditionally show this field, since it's only applicable to VPC load balancers:
```html
<div class="form-group" ng-if="loadBalancer.vpcId">
```
should do it